test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -26,8 +26,8 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.*" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.Emulator" Version="2.5.4" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.Redis" Version="0.1.9-alpha" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Emulator" Version="2.6.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Redis" Version="0.1.10-alpha" />
```

Comment:
Can these changes be reverted after we move to the newer DurableTask.Core update?